### ISSUE TITLE
Organize payloads by platform

### ISSUE THREAD

**Issue by ericfri** [hak5/bashbunny-payloads#79]
After looking through the payloads, it's not always imeediatly obvious which platform the payload is for.  I think it would make sense to organize them into Windows/Linux/Mac/Other folders.

**Comment by Biocow** [comment]
Agreed. I've been digging through what is there so far looking for the mac ones. Some of them are labeled, but some are not. Organized by folder might be cleaner.

**Comment by sebkinne** [comment]
After a conversation with @hak5darren, we will probably be reorganizing all payloads to match the following pattern: `/category/{win,osx,linux,multi}_NameOfPayload`. See examples below:

/exfiltration/win_smb_exfiltrator
/exfiltration/win_usb_exfiltrator
/exfiltration/osx_usb_SmacAndGrab
/exfiltration/multi_BunnyTap
/enumeration/osx_macinfograbber
/credentials/win_WiPassDump
/credentials/multi_QuickCreds
/prank/win_90sMode

**Comment by mrt0mat0** [comment]
why not fully adopt the msf structure and make the OS its own folder? also, should you maybe decide on coding standards for folders and such? camel case mixed with snake case is ugly. clean code is happy code :)

**Comment by rothgar** [comment]
+1 for top level folders instead of individual names.

**Comment by bg-wa** [comment]
I started this branch, based on conversation here:

https://github.com/bg-wa/bashbunny-payloads/tree/feature/issue_79

I hesitate to add top level folders for each platform, as I think the attack surface for this this device is much more diverse than than {win,osx,linux,multi}.  

No PR, just fleshing out an idea. All files are the just reorganized from `master`, except an example playload (`multi_hid_storage_example.txt`), I wrote to demo to a friend.

**Comment by mrt0mat0** [comment]
yes, but you almost always start your attack based on machine OS type. you
don't say, oh i want to steal this password, you say, oh i have a
linux/max/windows. what can i do to it with my bash bunny to take control?
Besides, the attack surface isn't any larger than MSF, is it?

**Comment by bg-wa** [comment]
Ha, I guess that does make more sense... It is not larger than MSF.

**Comment by Sneakypoke** [comment]
So I take it nothing happened with regards to the new structure?
